src/Schema/Blueprint.php

```diff
@@ -32,6 +31,26 @@ class Blueprint extends BaseBlueprint
 {
     use MarksAsNotSupported;
 
+    /**
+     * @inheritDoc
+     * @return IntColumnDefinition
+     */
+    public function bigInteger($column, $autoIncrement = false, $unsigned = false): IntColumnDefinition
```

Comment:
Is handling `$autoIncrement` and `$unsigned` unnecessary?

It would seem helpful to call `markAsNotSupported()`.

Reply:
Would be nice but I think that's out of scope from this PR.